prototypes/recipe.lua

```diff
@@ -25,9 +25,9 @@ data:extend({
 		ingredients =
 		{
 			{ type = "item", name = "warehouse-basic", amount = 1 },
-			{ type = "item", name = "passive-provider-chest", amount = 1 },
-			{ type = "item", name = "steel-plate", amount = 10 },
-			{ type = "item", name = "iron-stick", amount = 15 },
+			{ type = "item", name = "passive-provider-chest", amount = 5 },
+			{ type = "item", name = "steel-plate", amount = 25 },
```

Comment:
I had a chat with Nova (he's the one who initially planned out the mod with me)
How about a suggestion for:
- 1 of the logistic chests (for the port / logistics version)
- 25x advanced-circuit to fill out the requirements for the logistics stuff in the warehouse

This bolsters the copper requirement of the warehouses, which we both feel is a bit lacking at this stage, and would balance out the requirements

Also, requiring more steel doesn't make that much sense in lore, as your warehouse is already built, so you wouldn't need more steel

Reply:
That being said, we're both happy with the requirement of concrete for (as you said) the smoother floor for the logistics system

Reply:
I see that GitHub sees a conflict where local `git` can just adapt to the renamed files.

Would like to settle the recipes before I start rewriting any history, though.

And on that note, I have pushed my interpretation of how to apply advanced-circuit to logistic storehouses also (scaled down similarly to the concrete cost).